app/components/provider-links/style.scss

```diff
@@ -0,0 +1,42 @@
+& {
```

Comment:
Doesn't ember-component-css take care of making sure these styles only apply to this component? Do we need to wrap everything in `& {}`?

Reply:
It's not for scoping; it allows you to set css on component. So really the only thing that needs to go in `& {}` is the styling for the actual component. Should I pull everything else out?

Reply:
Ah, right, I skimmed right over the styles and just saw the classes. I guess it seems cleaner to use `&` only for the styles on the component element itself, and pull everything else out, but that's not a strong preference.

Reply:
@aaxelb, they have been un-nested.